.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please configure yourself a global gitignore on your workstation, and put these IDE-specific files and directories there instead.